tests/boot/boot_from_pxe.pm

```diff
@@ -61,29 +61,28 @@ sub run {
 
         my $openqa_url = get_required_var('OPENQA_URL');
         $openqa_url = 'http://' . $openqa_url unless $openqa_url =~ /http:\/\//;
-        my $repo        = $openqa_url . "/assets/repo/${image_name}";
-        my $sut_fqdn    = get_var('SUT_IP', 'nosutip');
-        my $sut_allowed = qr/(arch\.suse\.de|qa2\.suse\.asia)/im;
-        my $key_used    = '';
-        if (is_remote_backend && is_aarch64 && ($sut_fqdn =~ $sut_allowed)) {
+        my $repo     = $openqa_url . "/assets/repo/${image_name}";
+        my $key_used = '';
+        if (is_remote_backend && is_aarch64 && is_covered_machine) {
             $key_used = 'c';
             send_key 'down';
         }
         else {
             $key_used = 'esc';
         }
-        send_key_until_needlematch [qw(qa-net-boot orthos-grub-boot)], $key_used, 8, 3;
+        send_key_until_needlematch [qw(qa-net-boot orthos-grub-boot qa-net-grub-boot)], $key_used, 8, 3;
```

Comment:
Please add a comment for the new tag about what it is for.

Reply:
Done